orderer/common/localconfig/config_test.go

```diff
@@ -314,3 +314,17 @@ func TestChannelParticipationDefaults(t *testing.T) {
 	require.Equal(t, cfg.ChannelParticipation.Enabled, Defaults.ChannelParticipation.Enabled)
 	require.Equal(t, cfg.ChannelParticipation.MaxRequestBodySize, Defaults.ChannelParticipation.MaxRequestBodySize)
 }
+
+func TestKeepaliveDefaults(t *testing.T) {
```

Comment:
```suggestion
func TestKeepaliveConfig(t *testing.T) {
```

Actually, you are testing a config override, not just a default